eagleproject/core/blockchain/strategies.py

```diff
@@ -133,4 +134,11 @@
     "SUPPORTED_ASSETS": ["ETH", "OETH"],
     "ICON_NAME": "oeth-icon.svg",
   },
+  "morpho_aave_strat": {
```

Comment:
Let's prefix the key with "oeth_", I don't wanna take chances with a conflict in DB 😅